src/org/etools/j1939_84/bus/j1939/J1939TP.java

```diff
@@ -38,8 +38,6 @@
     final static public int T4 = 1050;
     final static public Map<Integer, String> table7;
     final static public int Th = 500;
-    final static public int Tr = 200;
```

Comment:
I'm surprised that Tr wasn't getting used.  It's a constant defined by the specification.  Instead, let's comment that the Tr value is enforced in J1939 reqeuest*Once calls.